flask/app/models.py

```diff
@@ -89,24 +88,23 @@ class ParticipantType(str, Enum):
     Proband = "Proband"
     Parent = "Parent"
     Sibling = "Sibling"
+    Other = "Other"
 
 
 @dataclass
 class Participant(db.Model):
     participant_id: int = db.Column(db.Integer, primary_key=True)
     family_id = db.Column(db.Integer, db.ForeignKey("family.family_id"), nullable=False)
-    # Sample.SampleName
     participant_codename: str = db.Column(db.String(50), nullable=False, unique=True)
-    # Sample.Gender
     sex: Sex = db.Column(db.Enum(Sex))
-    # Sample.SampleType
     participant_type: ParticipantType = db.Column(db.Enum(ParticipantType))
     month_of_birth: date = db.Column(
         db.Date, CheckConstraint("DAY(month_of_birth) = 1")
     )
-    # Sample.AffectedStatus
+    institution_id = db.Column(
+        db.Integer, db.ForeignKey("institution.institution_id", onupdate="cascade")
+    )
```

Comment:
Is this meant to be nullable? I was wondering why the tests weren't failing despite no changes.

Reply:
Hmm, good point. Currently it is nullable yes according to the metadata documentation. Since there's an 'Unknown' column though we _could_ make it be required